3ds Max/Max2Babylon/2015/Max2Babylon2015.csproj

```diff
@@ -309,7 +309,7 @@ if not exist "$(SolutionDir)assemblies\2015 - 2016" mkdir "$(SolutionDir)assembl
 del "$(SolutionDir)assemblies\2015 - 2016\*.dll"
 copy "$(TargetDir)*.dll" "$(SolutionDir)assemblies\2015 - 2016\"
 if exist "C:\Program Files\Autodesk\3ds Max 2016\bin\assemblies\" copy "$(SolutionDir)assemblies\2015 - 2016\*.dll" "C:\Program Files\Autodesk\3ds Max 2016\bin\assemblies\"
-if exist "D:\Programmes\Autodesk\3ds Max 2016\bin\assemblies\" copy "$(SolutionDir)assemblies\2015 - 2016\*.dll" "D:\Programmes\Autodesk\3ds Max 2016\bin\assemblies\"</PostBuildEvent>
+if exist "D:\Programmes\Autodesk\3ds Max 2015\bin\assemblies\" copy "$(SolutionDir)assemblies\2015 - 2016\*.dll" "D:\Programmes\Autodesk\3ds Max 2015\bin\assemblies\"</PostBuildEvent>
```

Comment:
We should do both to be at ease

Reply:
As you wish, paths are personal anyway, every contributor should add its own.

Reply:
yes but at least we wont change it on very PR :)
please merge after the change